src/coreclr/build-runtime.sh

```diff
@@ -109,6 +109,9 @@ __ConfigTriplet="$__TargetOS.$__TargetArch.$__BuildType"
 if [[ "$__TargetOS" == "linux-bionic" ]]; then
     __ConfigTriplet="linux.$__TargetArch.$__BuildType"
 fi
+if [[ ! -z "$__OutputRIDOS" ]]; then
+    __ConfigTriplet="$__OutputRIDOS.$__TargetArch.$__BuildType"
```

Comment:
Instead of introducing this override for output path, would it be better to keep the compiler in its default path and invoke it from there?

Reply:
In that case, the default path for the cross-compiler is `$(TargetOS).$(TargetArchitecture).$(Configuration)/$(BuildArchitecture)`, while the `jitinterface` and `clrjit` cross-components are generated in `$(HostOS).$(TargetArchitecture).$(Configuration)/$(BuildArchitecture)`, and these components cannot be found. 
https://github.com/dotnet/runtime/blob/36481bfd079093fe7d29ef024b1fb801e6c787fe/src/coreclr/tools/aot/ILCompiler/ILCompiler.props#L107


When building cross-components for cross-arch scenarios, the `DCLR_CMAKE_TARGET_ARCH` is used for the output path.
https://github.com/dotnet/runtime/blob/36481bfd079093fe7d29ef024b1fb801e6c787fe/src/coreclr/build-runtime.sh#L166-L168

One approach would be to introduce the `-hostos` parameter along with `-hostarch` to indicate the target platform for the cross-components build. Another approach would be to adjust the components path in the `_crossarch` projects.